dokka-subprojects/plugin-base/src/test/kotlin/model/TypesTest.kt

```diff
@@ -115,4 +118,26 @@ class TypesTest : AbstractModelTest("/src/main/kotlin/classes/Test.kt", "types")
             }
         }
     }
+
+    @Test
+    @OnlySymbols("context parameters")
+    @OptIn(ExperimentalDokkaApi::class)
+    fun `functional type with context parameters and receiver`() = withContextParametersEnabled {
```

Comment:
I see that there is a test above involving typealias to a functional type. Maybe it makes sense to also add similar test for context parameters?